.travis.yml

```diff
@@ -1,23 +0,0 @@
-language: bash
-services: docker
-
-env:
-  - VERSION=2.2 PYTHON_VERSION=3.6 PG_MAJOR=10 PG_VERSION=10.13-1.pgdg100+1 VARIANT=slim
-  - VERSION=2.2 PYTHON_VERSION=3.6 PG_MAJOR=12 PG_VERSION=12.2-r0 VARIANT=alpine
-  - VERSION=2.2 PYTHON_VERSION=3.7 PG_MAJOR=10 PG_VERSION=10.13-1.pgdg100+1 VARIANT=slim
-  - VERSION=2.2 PYTHON_VERSION=3.7 PG_MAJOR=12 PG_VERSION=12.2-r0 VARIANT=alpine
-  - VERSION=2.2 PYTHON_VERSION=3.8 PG_MAJOR=10 PG_VERSION=10.13-1.pgdg100+1 VARIANT=slim
-  - VERSION=2.2 PYTHON_VERSION=3.8 PG_MAJOR=12 PG_VERSION=12.2-r0 VARIANT=alpine
-
-before_script:
-  - wget -qO- 'https://github.com/tianon/pgp-happy-eyeballs/raw/master/hack-my-builds.sh' | bash
```

Comment:
I noticed that we didn't include this in `azavea/docker-tippecanoe`. I see that it's to make accessing PGP keyservers a little easier. Is this no longer beneficial?